solution.txt

```diff
@@ -107,8 +104,11 @@ Exercise 10:
 */
 
 // Complete Exercise 10 below...
+let hasSoup = true;
+if (foods.indexOf("soup") === -1 ) {
+  hasSoup = false;
+}
```

Comment:
amazing job using `const` and `let` appropriately! I know it's something I've said I personally prefer, but I appreciate you taking that detail into account 😄